proceeding/tables/survey_table.tex

```diff
@@ -0,0 +1,23 @@
+%table
+
+\begin{table*}[h]
+
+\caption{Survey information.}
```

Comment:
I would suggest "Survey" -> "Image" here and for the column "Survey name" -> "Image" below.
Or use "Survey image".

Usually the product of a survey (like Planck) is a set of images and catalogs, so just using "survey" as caption for the images isn't the best choice IMO.